private/gui-helpers.rkt

```diff
@@ -204,7 +209,10 @@
         (when (and (or is-directory
                        (not (xor (preferences:get 'files-viewer:filter-types2)
                                  (ormap (λ (x) (path-has-extension? i x)) filter-types))))
-                   (not (and (preferences:get 'files-viewer:filter-types3) (string-prefix? (path->string i) ".")))
+                   (not (and (preferences:get 'files-viewer:filter-types3)
+                             (string-prefix? (path->string i) ".")))
```

Comment:
I've got the editor preference for the 102 character vertical line turned on. Mine below was way over and the one above was as well so I broke both of them. I'm not going to go through and do style fixes as a general rule, not my code. Just figured I would explain the change.